packages/realm/bindgen/CMakeLists.txt

```diff
@@ -146,7 +147,7 @@ set(GENERATED_JS_FILES
 bindgen(
     TEMPLATE ${SDK_DIR}/bindgen/src/templates/node-wrapper.ts
     OUTPUTS ${GENERATED_JS_FILES}
-    OUTDIR ${TYPESCRIPT_OUTPUT_DIR}
+    OUTDIR ${GENERATED_OUTPUT_DIR}
```

Comment:
I think you meant to change lines 144/145 instead. Right now there are out of sync.